### ISSUE TITLE
Leave the keyframes on the current column/scene when collapse

### ISSUE THREAD

**Issue by F-Burning** [opentoonz/opentoonz#4616]
### Is your feature request related to a problem? Please describe

When you apply keyframes on a column and then collapse it, the keyframes will be collapsed too, and if you want to leave the keyframes on the current scene, you have to open the collapsed column and copy them, close and paste it.

### Describe the solution you'd like

The option to leave the column data (keyframes) on the actual scene when you collapse columns

**Comment by DarrenTAnims** [comment]
This could only work if you collapse 1 column, which isn't really what collapse is for, nor what it's useful for.
What if you collapse multiple columns that all have keyframes? (from the animate tool) Those animation keys only apply to 1 column, not all of them, so you couldn't really apply them to the whole subxsheet.
If you want the animation keys outside of the subxsheet, you're better off manually moving them to another column, then collapsing the appropriate column to a subxsheet, then moving them back onto the subxsheet.

**Comment by F-Burning** [comment]
That's true @DarrenTAnims, I forgot specify it. I mean collapse a single column, as it happens with precoms in After Effects (the utility is the same) When you precompose one layer, AE ask you if you want to leave the data on the current composition, or move it with the precom. I think this is useful to avoid the process I mentioned before every time you collapse a column.
